roles/matrix-bridge-mx-puppet-skype/defaults/main.yml

```diff
@@ -0,0 +1,205 @@
+# Mx Puppet Skype is a Matrix <-> Skype bridge
+# See: https://github.com/Sorunome/mx-puppet-skype
+
+matrix_mx_puppet_skype_enabled: true
+
+matrix_mx_puppet_skype_container_image_self_build: false
+
+matrix_mx_puppet_skype_docker_image: "sorunome/mx-puppet-skype:latest"
+matrix_mx_puppet_skype_docker_image_force_pull: "{{ matrix_mx_puppet_skype_docker_image.endswith(':latest') }}"
+
+matrix_mx_puppet_skype_base_path: "{{ matrix_base_data_path }}/mx-puppet-skype"
+matrix_mx_puppet_skype_config_path: "{{ matrix_mx_puppet_skype_base_path }}/data"
+matrix_mx_puppet_skype_data_path: "{{ matrix_mx_puppet_skype_base_path }}/data"
+matrix_mx_puppet_skype_docker_src_files_path: "{{ matrix_mx_puppet_skype_base_path }}/docker-src"
+
+matrix_mx_puppet_skype_appservice_port: "6000"
+
+matrix_mx_puppet_skype_homeserver_address: 'http://matrix-synapse:8008'
+matrix_mx_puppet_skype_homeserver_domain: '{{ matrix_domain }}'
+matrix_mx_puppet_skype_appservice_address: 'http://matrix-mx-puppet-skype:{{ matrix_mx_puppet_skype_appservice_port }}'
+
+# A list of extra arguments to pass to the container
+matrix_mx_puppet_skype_container_extra_arguments: []
+
+# List of systemd services that matrix-puppet-skype.service depends on.
+matrix_mx_puppet_skype_systemd_required_services_list: ['docker.service', 'matrix-synapse.service']
```

Comment:
Maybe it's better if you remove `'matrix-synapse.service'` from here and move it to `group_vars/matrix_servers` instead.

See how we do it for the Facebook bridge:

```yaml
matrix_mautrix_facebook_systemd_required_services_list: |
  {{
    ['docker.service']
    +
    (['matrix-synapse.service'] if matrix_synapse_enabled else [])
  }}
```

Reply:
Fixed!